sys/include/suit.h

```diff
@@ -62,6 +62,13 @@ extern "C" {
 #define CONFIG_SUIT_COMPONENT_MAX_NAME_LEN          (32U)
 #endif
 
+/**
+ * @brief Maximum length of a SUIT resource URL
+ */
+#ifndef SUIT_URL_MAX
+#define SUIT_URL_MAX            128
```

Comment:
Isn't this determined by `CONFIG_SOCK_URLPATH_MAXLEN`? Are they different?

Reply:
It also includes the host, so it should be longer.
I can make it depend on `CONFIG_SOCK_URLPATH_MAXLEN` though

Reply:
`makefiles/suit.inc.mk` already sets this to 128